database/seeds/OrdersSeeder.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use App\User;
+use App\Role;
+use App\Models\Product;
+
+class OrdersSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $customers = User::whereHas(
+            'roles', function($q){
+                $q->where('name', 'Customer');
+            }
+        )->get();
+        $faker = Faker\Factory::create();
+        foreach($customers as $customer)
+        {
+            $total_price = 0.0;
+            for($i=0; $i<5; $i++){
```

Comment:
sao lại < 5

Reply:
Dạ với mỗi customer thì em seed cho nó 1 cái order với 5 product, mỗi product thì em lấy random ra theo id á anh.

Reply:
sao lại dựa vào dữ liệu seed để fix cứng những chỗ này. 
phải dựa vào số product item mà người dùng order chứ. 
ví dụ: $i < order->product_item

Reply:
À, em xử lý quên attach nó vào :))). Ý em muốn là chạy vòng for tìm 4 thằng Product xong rồi add nó vào Order nên mới chạy vòng for vậy đó anh